src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,43 @@
+package duke;  //same package as the class being tested
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+    DTFormat dtf = new DTFormat();
+    Ui ui = new Ui(dtf);
+    TaskList tl = new TaskList();
+    Parser p = new Parser(dtf, ui, tl);
+
+    @Test
```

Comment:
The test dont seem to follow the naming convention 😢 It also seems like you may have commented out the test itself!